src/sass/_home.scss

```diff
@@ -252,22 +233,52 @@ h2{
       justify-content: center;
     }
   }
+  .left-button {
+    position: absolute;
+    top: 50%;
+    left: 0px;
+    z-index: 1;
+  }
+  .right-button {
+    position: absolute;
+    top: 50%;
+    right: 0px;
+    z-index: 1
+  }
+}
+.carusel {
+  position: relative;
+  margin: auto;
+  width: 840px;
+  overflow: hidden;
+  z-index: 0;
+  
+
+  }
+.pets-grid {
+  display: flex;
+  position: relative;
```

Comment:
add please `left: 0;`  - this will help transition to be applied on the first click